docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -1,8 +1,20 @@
 # trace_api_plugin
 
-## Description
+## Overview
 
-The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin defines a new HTTP endpoint accessible (see the [API reference](api-reference/index.md) for more information).
+The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin stores serialized block trace data to the filesystem for later retrieval via HTTP RPC requests.
```

Comment:
can you pls add an issue "add 'retired action' to glossary"?

Reply:
Sure thing